sdks/java/core/src/main/java/org/apache/beam/sdk/options/PipelineOptions.java

```diff
@@ -349,4 +349,9 @@ public Long create(PipelineOptions options) {
       return NEXT_ID.getAndIncrement();
     }
   }
+
+  @Description("Enable/disable metrics reporting.")
```

Comment:
What about runners that provide their own metrics (eg., number of elements processed)? Should this disable those as well? Why would a user want to disable metrics reporting?